### PR TITLE
made the normaL check up page mobile responsive

### LINKED ISSUE #1002: 🐞[Bug]:  normal check up page is not mobile responsive
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

normal check up page is not mobile responsive

### Expected behavior

it should be responsive for mobile and tablet screens

### Add ScreenShots

_No response_

### What browsers are you seeing the problem on?

_No response_

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by samarsajad**
## Description
made the normaL CHECK UPS PAGE mobile RESPONSIVE

## Related Issues

- Closes #1002 

## Type of PR

- [x] (Bug Fix)

## Checklist

- [x] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [x] I have performed a self-review of my code
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x

https://github.com/user-attachments/assets/7ecaeb4c-15ee-4e34-bc19-6fc7286e95da

] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by samarsajad** [comment]
Please review my pull request @varshith257

File: index.css [hunk 1]
```diff
@@ -700,7 +700,7 @@ body {
 }
 
 
-@media (max-width:768px) {
+/* @media (max-width:768px) {
```
Comment (varshith257) [inline index.css:703]: Why this commented?

**Comment by varshith257** [comment]
@Anjaliavv51 Take a look. I can see in deploy whole index.html is misaligned colliding items one upon one

**Comment by Anjaliavv51** [comment]
@samarsajad please close this PR and also create a new branch , and do the changes 

Screenshot_20240730_083750_Chrome

Screenshot_20240730_083745_Chrome

Screenshot_20240730_083736_Chrome

**Comment by Anjaliavv51** [comment]
@samarsajad we can't let to effect the main repo due to these changes soo pls create new branch and also close this PR too.

**Pull request closed without merging**